<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesContents</class>
 <widget class="QWidget" name="PreferencesContents">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="themeLabel">
       <property name="text">
        <string>Theme:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="themeComboBox"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QCheckBox" name="splashEnabledCheckBox">
       <property name="text">
        <string>Show splash screen on startup</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="splashSecondsLabel">
       <property name="text">
        <string>Splash screen duration:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="splashSecondsSpinBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="suffix">
        <string> seconds</string>
       </property>
       <property name="specialValueText">
        <string>Until OK clicked</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="rcloneDebugCheckBox">
       <property name="text">
        <string>Enable rclone debug logging</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="resetToDefaultsButton">
     <property name="text">
      <string>Reset to Defaults</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
